server/src/common/utils/uaiUtils.js

```diff
@@ -0,0 +1,40 @@
+const ALPHABET_23_LETTERS = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "p",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+
+module.exports = {
+  validateUAI: (code) => {
```

Comment:
Si tu veux, u peux utiliser la syntaxe raccourcie :
 ```suggestion
  validateUAI(code) {
```